vertexai/multimodal-video/multimodalvideo_test.go

```diff
@@ -25,12 +25,10 @@ func Test_generateMultimodalContent(t *testing.T) {
 	tc := testutil.SystemTest(t)
 
 	buf := new(bytes.Buffer)
-	prompt := "What is in this video?"
-	video := "gs://cloud-samples-data/video/animals.mp4"
 	location := "us-central1"
 	modelName := "gemini-1.5-flash-001"
 
-	err := generateMultimodalContent(buf, prompt, video, tc.ProjectID, location, modelName)
+	err := generateMultimodalContent(buf, tc.ProjectID, location, modelName)
```

Comment:
fix: please update the test to check the generation response ([example](https://github.com/GoogleCloudPlatform/golang-samples/blob/main/vertexai/multimodal-video-with-audio/multimodalvideoaudio_test.go#L42-L52))